package.json

```diff
@@ -1,60 +1,28 @@
 {
   "name": "scratch-blocks",
-  "version": "1.1.86",
```

Comment:
I'm slightly worried that bumping this will mess up their semantic release later. (like it'll bump to 3.0.0 when we go to release). Could you file an issue to check on this when we do actual integration into their repo? (or I can once I get issue filing permissions)

Reply:
Filed #17 to track.